src/shared/regions/defaultRegionProvider.ts

```diff
@@ -73,6 +74,15 @@ export class DefaultRegionProvider implements RegionProvider {
     }
 }
 
+export function makeEndpointsResourceFetcher(extensionContext: vscode.ExtensionContext): ResourceFetcher {
```

Comment:
This is essentially the same code as makeSampleRequestManifestResourceFetcher, would it make sense to have a dedicated utility function to create a CompositeResourceFetcher?

Reply:
There is a good chance that their use cases will diverge in the near future (for example, Region data may take an approach where it loads local data and uses it until remote data can be obtained). I'd like to leave these as part of their respective files for this iteration.